<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientQuotaDiscussionEditor</class>
 <widget class="QDialog" name="ClientQuotaDiscussionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDateMessage">
     <property name="text">
      <string>Дата и время сообщения</string>
     </property>
     <property name="buddy">
      <cstring>edtDateMessage</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateTimeEdit" name="edtDateMessage">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>115</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblAgreementType">
     <property name="text">
      <string>Тип согласования</string>
     </property>
     <property name="buddy">
      <cstring>cmbAgreementType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbAgreementType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblResponsiblePerson">
     <property name="text">
      <string>Ответственный ЛПУ</string>
     </property>
     <property name="buddy">
      <cstring>cmbResponsiblePerson</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbResponsiblePerson"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblCosignatory">
     <property name="text">
      <string>Контрагент</string>
     </property>
     <property name="buddy">
      <cstring>edtCosignatory</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="edtCosignatory"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblCosignatoryPost">
     <property name="text">
      <string>Должность</string>
     </property>
     <property name="buddy">
      <cstring>edtCosignatoryPost</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="edtCosignatoryPost"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblCosignatoryName">
     <property name="text">
      <string>ФИО</string>
     </property>
     <property name="buddy">
      <cstring>edtCosignatoryName</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="edtCosignatoryName"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblRemark">
     <property name="text">
      <string>Примечания</string>
     </property>
     <property name="buddy">
      <cstring>edtRemark</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" rowspan="2" colspan="2">
    <widget class="QTextEdit" name="edtRemark"/>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>79</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtDateMessage</tabstop>
  <tabstop>cmbAgreementType</tabstop>
  <tabstop>cmbResponsiblePerson</tabstop>
  <tabstop>edtCosignatory</tabstop>
  <tabstop>edtCosignatoryPost</tabstop>
  <tabstop>edtCosignatoryName</tabstop>
  <tabstop>edtRemark</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClientQuotaDiscussionEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClientQuotaDiscussionEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
